### DISCUSSION TITLE
Ionos domain redirecting to Vercel website does not load mobile version, but works fine for desktop version

### DISCUSSION THREAD

**Discussion by Leslie23971** [vercel/community#2855] [Help] [answered]
### Summary

Hi,

Title hopefully is self explanatory, but following are further details on my workflow here :
- website implemented with Nuxt.js
- Built and deployed via Vercel
- Domain hosted by Ionos

When using Vercel addresses/links everything works fine for both desktop and mobile version, but when using my Ionos domain to access same website via domain redirection, desktop version works fine, but mobile version does not load, I still see desktop version on any mobile I use when using domain address instead.

Ionos says it's not on their side, it's an issue with website builder...

Does anyone have any idea what I am missing here ?
(I am kind of a beginner when it comes to web dev...)

Thanks in advance for an help.

### Steps to Reproduce

_launch website..._ (not sure I should share my website address here)

**Comment by smaeda-ks** [comment]
@Leslie23971 what's the domain in question?

**Reply by Leslie23971** [reply]
see closing comment, thanks for helping

**Comment by Leslie23971** [comment, marked as answer]
Edit/Update : fixed, I wasn't linking my domain the right way, had set up a redirection from domain hoster instead of setting correct DNS on hoster side and setting domain in Vercel :/ - bear with me, beginner mistake, fixed now !
